local/dfm/cmd/install.sh

```
@@ -0,0 +1,38 @@
+# Installation functions for dfm, the dotfile manager
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)